src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNotesByOrganisationRequestHandler.cs

```diff
@@ -51,7 +51,7 @@ public async Task<EvidenceNoteSearchDataResult> HandleAsync(GetEvidenceNotesByOr
                 recipientId = null;
             }
 
-            var filter = new NoteFilter(request.ComplianceYear, int.MaxValue, 1)
+            var filter = new NoteFilter(request.ComplianceYear, request.PageSize, request.PageNumber)
```

Comment:
If not already done update the unit test to check the data access is called with the correct page and page size